packages/core/data-transfer/src/strapi/queries/link.ts

```diff
@@ -4,7 +4,7 @@ import { ILink } from '../../../types';
 
 // TODO: Remove any types when we'll have types for DB metadata
 
-export const createLinkQuery = (strapi: Strapi.Strapi) => {
+export const createLinkQuery = (strapi: Strapi.Strapi, trx?: any) => {
```

Comment:
We need smth else than any here.
You can install knex as a devDependency to get access to its types. (just like we did for koa)